src/stylesheets/fonts.scss

```diff
@@ -38,55 +38,55 @@
 
 /* 300 - Light */
 @font-face {
-    font-family: 'Noto Sans';
+    font-family: "Noto Sans";
     font-style: normal;
     font-weight: 300;
-    src: url('../fonts/NotoSans-Light.woff') format('woff');
-    src: url('../fonts/NotoSans-Light.woff2') format('woff2'); 
+    src: url("../fonts/NotoSans-Light.woff") format("woff");
+    src: url("../fonts/NotoSans-Light.woff2") format("woff2");
 }
 @font-face {
-    font-family: 'Noto Sans';
+    font-family: "Noto Sans";
     font-style: italic;
     font-weight: 300;
-    src: url('../fonts/NotoSans-LightItalic.woff') format('woff');
-    src: url('../fonts/NotoSans-LightItalic.woff2') format('woff2'),
+    src: url("../fonts/NotoSans-LightItalic.woff") format("woff");
+    src: url("../fonts/NotoSans-LightItalic.woff2") format("woff2");
 }
 
 /* 400 -  Regular */
 @font-face {
-    font-family: 'Noto Sans';
+    font-family: "Noto Sans";
     font-style: normal;
     font-weight: normal;
-    src: url('../fonts/NotoSans-Regular.woff') format('woff');
-    src: url('../fonts/NotoSans-Regular.woff2') format('woff2');
+    src: url("../fonts/NotoSans-Regular.woff") format("woff");
+    src: url("../fonts/NotoSans-Regular.woff2") format("woff2");
 }
 @font-face {
-    font-family: 'Noto Sans';
+    font-family: "Noto Sans";
     font-style: italic;
     font-weight: normal;
-    src: url('../fonts/NotoSans-Italic.woff') format('woff');
-    src: url('../fonts/NotoSans-Italic.woff2') format('woff2');
+    src: url("../fonts/NotoSans-Italic.woff") format("woff");
+    src: url("../fonts/NotoSans-Italic.woff2") format("woff2");
```

Comment:
Single quote is more in line with our other style guides. Better for the muscle memory.

Reply:
That's for not-SCSS, this is SCSS. Right?

Reply:
Perfectly reasonable for me, but maybe slightly tenuous for others explanation:

HTML attributes are quoted with double quotes. I'd like to see those double quotes again in selectors like `input[type="password"]` and once you're using them there… mixing them with single quotes is yuck.

Reply:
Yes. But we intentionally try to keep things as similar across languages as possible to avoid constantly writing the wrong thing out of muscle memory. SCSS and JS will likely be written by the same people so there's also no reason to arbitrarily change between the languages.

Reply:
I don't think this is arbitrary. CSS closer to the HTML that it works on than it is to JS. As for the muscle memory argument… in TSX we have to type single quotes, double quotes and backticks.

Reply:
Well this was one of the things Imre and Alan and I discussed with the web guys way back and we agreed that we'd use single quotes whenever possible in all front-end code. I can't think of any situation where double quotes are not a violation of the linting rules. And you can't make the "SCSS is closer to HTML" argument; it's a separate language just like JS and HTML.

Reply:
I agree with Ian on this one.

Reply:
Changed all double quotes to single quotes. Going to cry in a corner now.

Reply:
_hugs_